.gitmodules

```diff
@@ -1,6 +1,6 @@
 [submodule "spack"]
   path = spack
-  #url = https://github.com/spack/spack
-  #branch = develop
-  url = https://github.com/NOAA-EMC/spack
-  branch = jcsda_emc_spack_stack
+  url = https://github.com/kgerheiser/spack.git
```

Comment:
The `.git` is coming back! Any reason we need that?

Reply:
No, I didn't know about that.